docs/concepts/queries.md

```diff
@@ -0,0 +1,51 @@
+---
+id: queries
+title: Queries
+sidebar_label: Queries
+hide_title: true
+---
+
+# `Queries`
+
+This is the most basic feature of `rtk-query`. A `query` can be performed with any data fetching library of your choice and will accept any HTTP method, but we recommend that you only use queries for `GET` requests. For anything that alters data on the server or will possibly invalidate the cache, you should use a [Mutation](./mutations).
```

Comment:
I wouldn't keep this too focused on http here. 
Theroetically, these could also be used to make grpc calls or anything else, really.